handlers/routes.go

```diff
@@ -0,0 +1,20 @@
+package handlers
+
+func (s *Server) InitializeRoutes() {
```

Comment:
This method as well. How about `InitRoutes` or `HandleRoutes`? Let me know your thoughts.

Reply:
I agree with you, I'll rename to InitRoutes